server/core/src/main/scala/sttp/tapir/server/interceptor/log/ServerLog.scala

```diff
@@ -42,6 +42,10 @@ trait ServerLog[F[_]] {
 
   /** Invoked when an exception has been thrown when running the server logic or handling decode failures. */
   def exception(ctx: ExceptionContext[_, _], ex: Throwable, token: TOKEN): F[Unit]
+
+  /** Allows defining a list of endpoints which should not log requestHandled. Exceptions, decode failures and security failures will still be logged.
+    */
+  def ignoreEndpoints: Seq[AnyEndpoint] = Seq.empty
```

Comment:
since we are doing a `.contains` test, shouldn't this be a set?

Reply:
`Set` is invariant, which makes failed compilation for calls like
```scala
.ignoreEndpoints(Set(endpoint1, endpoint2))
```
and requires (not very obvious) update to
```scala
.ignoreEndpoints(Set[AnyEndpoint](endpoint1, endpoint2))
```
which may be confusing for library users?

The compilation error is
```
[error] /home/.../tapir/server/tests/ServerOptionsTests.scala:73:10: 
overloaded method value ignoreEndpoints with alternatives:
[error]   (es: Set[sttp.tapir.AnyEndpoint])sttp.tapir.server.interceptor.log.DefaultServerLog[F] <and>
[error]   => Set[sttp.tapir.AnyEndpoint]
[error]  cannot be applied to (scala.collection.immutable.Set[sttp.tapir.Endpoint[Unit,Unit,Unit,String,Any]])
[error]         .ignoreEndpoints(Set(silentEndpoint1, silentEndpoint2))
```

Reply:
Hm so let's convert to a set inside the `.ignoreEndpoints` implementation?